### PR TITLE
Added JFixtures#compile method which returns result with instructions

### PR DESCRIPTION
**Pull request by vkorobkov**
The implementation of https://github.com/vkorobkov/jfixtures/issues/168

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -51,7 +51,7 @@
 
         <!--Test dependencies versions-->
         <spock-core.version>1.1-groovy-2.4</spock-core.version>
-        <equalsverifier.version>2.4.3</equalsverifier.version>
+        <equalsverifier.version>2.4.5</equalsverifier.version>
```
Comment (rodionovsasha) [inline pom.xml]: Should not be a different PR? :) :)

File: pom.xml [hunk 2]
```diff
@@ -184,6 +184,10 @@
                         <exclude>com/github/vkorobkov/jfixtures/fluent/impl/JFixturesResultBase*.class</exclude>
                         <!--test exception for asString method-->
                         <exclude>com/github/vkorobkov/jfixtures/fluent/impl/XmlJFixturesResultImpl*.class</exclude>
+                        <!--Impossible to fully test auto generated <equals> method with the latest lombok-->
+                        <exclude>com/github/vkorobkov/jfixtures/instructions/CleanTable.class</exclude>
+                        <exclude>com/github/vkorobkov/jfixtures/instructions/CustomSql.class</exclude>
+                        <exclude>com/github/vkorobkov/jfixtures/instructions/InsertRow.class</exclude>
```
Comment (rodionovsasha) [inline pom.xml:190]: :(

File: src/main/java/com/github/vkorobkov/jfixtures/JFixtures.java [hunk 3]
```diff
@@ -48,4 +50,17 @@ public JFixtures addTables(Collection<Table> newTables) {
             CollectionUtil.concat(this.tables, newTables)
         );
     }
+
+    public Result compile() {
+        List<Instruction> instructions = new Processor(
+            Table.mergeTables(tables),
+            loadConfig()
```
Comment (rodionovsasha) [inline src/main/java/com/github/vkorobkov/jfixtures/JFixtures.java]: Move to line above?

File: src/main/java/com/github/vkorobkov/jfixtures/config/ConfigLoader.java [hunk 4]
```diff
@@ -3,24 +3,19 @@
...
     }
 
-    private Node loadNode(Path path) {
-        if (Files.exists(path)) {
-            return Node.root(YmlUtil.load(path));
-        }
-        log.info("File '" + path + "' not found, using default configuration");
-        return Node.emptyRoot();
+    public Root defaultConfig() {
+        return new Root(Node.emptyRoot());
```
Comment (rodionovsasha) [inline src/main/java/com/github/vkorobkov/jfixtures/config/ConfigLoader.java:19]: 👍

**Review by rodionovsasha — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (vkorobkov) [reply pom.xml]: Ok, I'll revert it.
